content/en/tests/early_flake_detection.md

```diff
@@ -0,0 +1,117 @@
+
+Early Flake Detection normally works as follows:
+
+* A developer working in a feature branch writes a new test, commits and pushes the changes.
+* This test is retried automatically for this commit and every new commit in the feature branch, until the branch is merged.
+* After the feature branch is merged, that new test is considered part of the known tests for the test service. The test is no longer treated as new.
+
+{{< img src="continuous_integration/early_flake_detection_commits.png" alt="How Early Flake Detection works in your commits.">}}
+
+### Excluded Branches from Early Flake Detection
+
+The Excluded Branches will not have any of their tests retried by Early Flake Detection.
+
+Additionally, what Early Flake Detection considers a new test is based on the tests that run in these branches. If a test has run in any of the Excluded Branches from Early Flake Detection, it is **not** considered new anymore.
+
+## Set up a Datadog library
+Before setting up Early Flake Detection, you must configure [Test Visibility][5] for your particular language. If you are reporting data through the Agent, use v6.40 or 7.40 and later.
+
+## Configuration
+After you have set up your Datadog library for Test Visibility, configure Early Flake Detection from the [Test Service Settings page][6]:
+
+{{< img src="continuous_integration/early_flake_detection_test_settings.png" alt="Early flake Detection in Test Service Settings.">}}
+
+Click on **Configure** in the Early Flake Detection column to see this modal:
+
+{{< img src="continuous_integration/early_flake_detection_configuration_modal.png" alt="Early flake Detection configuration modal.">}}
+
+In this modal you can toggle the status of Early Flake Detection and add a list of [**Excluded Branches from Early Flake Detection**][4] for that test service.
+
+### How to check that new tests are detected
+
+There are facets to query sessions that run Early Flake Detection and new tests.
+
+#### Test session
+
+Test sessions running Early Flake Detection have the `@test.early_flake.enabled` tag set to `true`.
+
+#### Test
+
+New tests have the `@test.is_new` tag set to `true`.
+
+Additionally, retries for this test have the `@test.is_retry` tag set to `true`.
+
+
+## Troubleshooting
+
+If you suspect there are issues with Early Flake Detection, go to the [Test Service Settings][6], look for your test service, and click **Configure**. Disable Early Flake Detection by clicking on the toggle.
+
+### A new test is not being retried
+
+This could be caused by a couple of reasons:
+
+* This test has already run in an excluded branch, such as `staging`, `main` or `preprod`.
+* This test is slower than 5 minutes. There is a mechanism not to run Early Flake Detection on tests that are too slow, since retrying these tests could cause significant delays in CI pipelines.
```

Comment:
```suggestion
* This test has already run in an excluded branch, such as `staging`, `main`, or `preprod`.
* This test is slower than five minutes. There is a mechanism not to run Early Flake Detection on tests that are too slow, since retrying these tests could cause significant delays in CI pipelines.
```